cypress/integration/business-pages.test.js

```diff
@@ -0,0 +1,130 @@
+/// <reference types="cypress" />
+import { config } from '../../src/configs';
+
+describe('Business pages test ', () => {
+  let pageCode;
+  let ukHeader;
```

Comment:
Good job on testing with Cypress! As I remember @vic0803 sad that we should use "ua" in the Ukrainian language description and mentions because "uk" is "United Kingdom"
 ```suggestion
  let uaHeader;
```

Reply:
fixed